test/library/packages/LinearAlgebra/correctness/testSparseTranspose.chpl

```diff
@@ -0,0 +1,81 @@
+use LinearAlgebra;
+use LinearAlgebra.Sparse;
+use BlockDist;
+use Lists;
```

Comment:
This module was renamed back to `List` (sorry for the back-and-forth). Changing it to `List` makes this test pass on master. I will merge it once it is changed. (Please keep your change limited to that, as I don't want to run more large tests on this)